WebAPI/Services/User/Authentication/UserAuthenticationService.cs

```diff
@@ -52,17 +52,17 @@ public async Task<Response> Login(LoginModel model)
                     );
 
                 var tokenResponse = new JwtSecurityTokenHandler().WriteToken(token);
+                bool _isAdmin = await _userManager.IsInRoleAsync(user, "Admin");
 
-                return new Response { StatusCode = 200, Message = $"{tokenResponse}" };
+                return new Response { StatusCode = 200, Message = $"{tokenResponse}", isAdmin = _isAdmin };
```

Comment:
Move StatusCodes to some enum for easier acces in future. You can also use HttpStatusCode built-in enum